build.sh

```diff
@@ -111,7 +111,11 @@ EOF
 file_names=`git diff --name-only $TRAVIS_COMMIT_RANGE`
 
 for f in $file_names; do
-   # filter files
+  if [ ! -f "$f" ]; then
```

Comment:
Should be -e "$f" or else will fail if $f is a symbolic link --- unless we have a policy of no symbolic links, ever.